roles/gateway_common/tasks/initial-update-root-password.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: Update expired root user password
-  no_log: true # don't log decrypted passwords in responses
+  no_log: "{{ no_log }}" # don't log decrypted passwords in responses
```

Comment:
@david-dc-chen I did this with another variable and Ansible had a stack overflow error. This one works?

Reply:
I tried with gateway-autoprovision-nodes.yml  and it works there. I've used {{ no_log_flag }} in the past and it worked too at time.

Reply:
Hmm ok, might still wanna pick a different variable name though. It's a bit confusing as a reader to see `x = x`.